source/content/mu-plugin.md

```diff
@@ -239,7 +239,7 @@ We can add multiple plugins to the function as well:
 wp_cache_add_non_persistent_groups( array( 'my_plugin_group', 'woocommerce' ) );
 ```
 
-To verify, you can use the [Redis CLI](/redis#use-the-redis-command-line-client) to flush all keys and see that the related objects are no longer added to the cache:
+To verify, you can use the [Redis CLI](/object-cache#use-the-redis-command-line-client) to flush all keys and see that the related objects are no longer added to the cache:
```

Comment:
is the command line client being renamed as well?

Reply:
No. It is the Redis command line client. If the client or its name change, it will happen later. For now this is accurate.